autogen/oai/client.py

```diff
@@ -346,16 +366,14 @@ def _completions_create(self, client, params):
                         index=i,
                         finish_reason=finish_reasons[i],
                         message=ChatCompletionMessage(
-                            role="assistant", content=response_contents[i], function_call=None
+                            role="assistant", content=response_contents[i], function_call=full_function_call
                         ),
                     )
 
                 response.choices.append(choice)
         else:
-            # If streaming is not enabled or using functions, send a regular chat completion request
-            # Functions are not supported, so ensure streaming is disabled
+            # If streaming is not enabled, send a regular chat completion request
             params = params.copy()
-            params["stream"] = False
```

Comment:
Restoring this line makes the failing test pass. Please restore it and we are ready to go.